lesson_2/fizzbuzz.py

```diff
@@ -0,0 +1,20 @@
+"""
+Prob 2: Fizz Buzz
+
+-Write a program that prints the numbers from 1 to 100 inclusive.
+-But for multiples of three print “Fizz” instead of the number.
+-For the multiples of five print “Buzz” instead of the number.
+-For numbers which are multiples of both three and five print “FizzBuzz” instead. 
+
+"""
+
+
+for x in range(1,101):
+    if x % 3 == 0 and x % 5 == 0:
+        print("FizzBuzz")
+    elif x % 3 == 0 and x % 5 != 0:
+        print("Fizz")
+    elif x % 3 != 0 and x % 5 == 0:
+        print("Buzz")
+    else:
+        print(x)
```

Comment:
your `if` statement at line 13 already checks for being divisible by both 3 and 5 so `elif` checks on lines 15 and 17 don't need to check both.

meaning:
```
if x % 3 == 0 and x % 5 == 0
elif x % 3 == 0
elif x % 5 == 0
```
are sufficient